### PR TITLE
Update Address__c.object

### PR DESCRIPTION
**Pull request by searheart**
# Critical Changes

# Changes
Added four fields to the address object in support of Insights Address Services release

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

**Comment by lmeerkatz** [comment]
@searheart Can you clone the NPSP repo and branch from there? I believe that will give you automatic merges from master, which will make development easier.

**Comment by force2b** [comment]
@searheart It would be helpful if you could do that instead of forking the repo. Also, feel free to assign me as the reviewer/assignee for the PR.

Other questions:

- Should one or more of these new fields be on the default Address page layout for all new installs? If so, can you either include an updated layout file or maybe provide a screen capture of the address record showing where you want the new fields.?
- You had mentioned apex changes that went along with the new fields. Are they coming in a separate PR?

**Comment by searheart** [comment]
@force2b The apex work and the page layout will be in a different PR.  I need to get UX to look at the address page layout to approve my suggested changes.

**Comment by lmeerkatz** [comment]
@searheart Let's get this moved into a new branch now that you have repo access. I would want to hold these changes until the Apex work is ready if we can, as anything merged into master is eligible for release and I wouldn't want to release the fields until the supporting automation is ready. /cc @force2b

File: src/objects/Address__c-Address Layout.layout [hunk 1]
```diff
@@ -0,0 +1,196 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (force2b) [inline src/objects/Address__c-Address Layout.layout]: Thanks, though this file needs to be in the src/layouts folder. You'll see there is already a layout file there. That file just needs to be updated to include the new fields.

File: src/objects/Address__c.object [hunk 2]
```diff
@@ -4,40 +4,140 @@
         <actionName>Accept</actionName>
         <type>Default</type>
     </actionOverrides>
+    <actionOverrides>
```
Comment (force2b) [inline src/objects/Address__c.object]: Nothing should be changed in this file other than the new fields.

File: src/objects/package.xml [hunk 3]
```diff
@@ -0,0 +1,3596 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (force2b) [inline src/objects/package.xml]: As with the layout file, there is already an existing package.xml in the src folder. That file should only have 4 changed lines for the 4 new fields.

**Comment by force2b** [comment]
@searheart  Also, now that you have access to create a branch in our repo can you close this PR, clone the npsp repo locally, create a new branch and then a new PR with the changes? This is still a forked PR so it cannot be merged.

**Pull request closed without merging**